src/Monolog/Logger.php

```diff
@@ -697,4 +728,28 @@ public static function setTimezone(\DateTimeZone $tz)
     {
         self::$timezone = $tz;
     }
+
+    /**
+     * Set the minimal log level.
+     *
+     * Logger will log only records with log level >= $level.
+     *
+     * @param string $level
+     */
+    public function setLevel($level) {
+        if (!isset(static::$levelMapping[strtoupper($level)])) {
```

Comment:
The `strtoupper($level)` meets twice per method and it seems reasonable to store the result in a variable.

Reply:
Fixed.